src/configuration.rs

```diff
@@ -0,0 +1,8 @@
+use clap::Parser;
```

Comment:
Nit: Needs Copyright

```suggestion
/// Copyright 2022 MobileCoin Foundation

use clap::Parser;
```